core/jobs.py

```diff
@@ -167,9 +167,9 @@ def register_start(cls, job_id, metadata=None):
         cls._require_valid_transition(
             job_id, model.status_code, STATUS_CODE_STARTED)
         cls._require_correct_job_type(model.job_type)
-
+        """_pre_start_hook is a null block which takes in ID of the the job to 
```

Comment:
Place the docstrings in the correct position. Ditto elsewhere. Please see already documented files if you are facing any problems.